pkg/alert/alertservice.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright (C) 2018 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownershia. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package alert
+
+import (
+	"fmt"
+	"strconv"
+
+	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
+	"github.com/blackducksoftware/horizon/pkg/components"
+)
+
+// getAlertService returns a new Service for an Alert
+func (a *SpecConfig) getAlertService() *components.Service {
+	service := components.NewService(horizonapi.ServiceConfig{
+		Name:          "alert",
+		Namespace:     a.config.Namespace,
+		IPServiceType: horizonapi.ClusterIPServiceTypeNodePort,
+	})
+
+	service.AddPort(horizonapi.ServicePortConfig{
+		Port:       int32(*a.config.Port),
+		TargetPort: strconv.Itoa(*a.config.Port),
+		Protocol:   horizonapi.ProtocolTCP,
+		Name:       fmt.Sprintf("%s-tcp", strconv.Itoa(*a.config.Port)),
+	})
+
+	service.AddSelectors(map[string]string{"app": "alert"})
+	service.AddLabels(map[string]string{"app": "alert"})
+	return service
+}
+
+// getAlertExposedService returns a new Exposed Service for an Alert
+func (a *SpecConfig) getAlertExposedService() *components.Service {
+	service := components.NewService(horizonapi.ServiceConfig{
+		Name:          "alert-lb",
+		Namespace:     a.config.Namespace,
+		IPServiceType: horizonapi.ClusterIPServiceTypeLoadBalancer,
+	})
+
+	service.AddPort(horizonapi.ServicePortConfig{
+		Port:       int32(*a.config.Port),
+		TargetPort: strconv.Itoa(*a.config.Port),
```

Comment:
Probably can use `fmt.Sprintf` here as well and remove the need for importing `strconv`

Reply:
Good idea